--- perl/perl.bzl
+++ perl/perl.bzl
@@ -169,7 +169,7 @@ def _perl_binary_implementation(ctx):
 
     return DefaultInfo(
         files = depset([ctx.outputs.executable]),
-        default_runfiles = ctx.runfiles(
+        runfiles = ctx.runfiles(
             collect_data = True,
             collect_default = True,
             transitive_files = depset([ctx.outputs.executable], transitive = [transitive_sources]),
